### DISCUSSION TITLE
revokeAchievement method

### DISCUSSION THREAD

**Discussion by WimDeMeester** [cjmellor/level-up#81] [Ideas]
Is it possible to add a method revokeAchievement?  
In my project, I only work with achievements.  I'm working on a logging program for astronomical observations.  When an observer has observed 25 objects (from a certain catalog), a bronze medal is awarded.  When 50 objects are observed, a silver medal is awarded, but the bronze medal should be removed.  Is that possible to do at this moment, or do we need a revokeAchievement method for this?

**Comment by cjmellor** [comment]
Cool idea. If I can find some time I can look at implementing it, but I'm also open to a PR too.

**Comment by cjmellor** [comment]
This has been implemented
